config/betterquesting/DefaultQuests.json

```diff
@@ -7666,7 +7683,7 @@
           "lockedProgress:1": 0,
           "autoClaim:1": 0,
           "isSilent:1": 0,
-          "desc:8": "You need steel? Uhh... fine. Let\u0027s get busy...\n\nHint: The bricked blast furnace can share walls with other ones, just like most GT multi blocks. How will this affect the air quality though...\n\n§3If you are short on Stone Dust, you can get more by macerating cobblestone.\n\n§4You\u0027ll need 432 unfired clay brick, 432 brick, 108 clay and brick dust, 216 compressed fireclay, and 216 firebrick. Collect all the required materials for these first, or you\u0027ll have to do it again. Also, don\u0027t make the firebrick blocks until you collect all the gypsum you need unless you want to be forced to get more."
+          "desc:8": "You need steel? Uhh... fine. Let\u0027s get busy...\n\nHint: The bricked blast furnace can share walls with other ones, just like most GT multi blocks. How will this affect the air quality though...\n\n§3If you are short on Stone Dust, you can get more by macerating cobblestone.\n\nIf you spawn near a biome with Hardened Clay you can macerate it to get the clay dust you need. You can also use brick blocks from Villages to get brick dust in the macerator.\n\nIf you can, craft a WorkTable from Forestry it will make the crafting a lot easier as you can craft with all your buckets in one click\n\n§4You\u0027ll need 432 unfired clay brick, 432 brick, 108 clay and brick dust, 216 compressed fireclay, and 216 firebrick. Collect all the required materials for these first, or you\u0027ll have to do it again. Also, don\u0027t make the firebrick blocks until you collect all the gypsum you need unless you want to be forced to get more."
```

Comment:
The comment text change is very good (though I'd word the sentence as:

Crafting the WorkTable from Forestry can make it easier as you can use all the buckets in one click.

Reply:
Great